app/src/main/java/com/example/househomey/Item.java

```diff
@@ -14,10 +17,10 @@
 
 /**
  * This class represents an inventory item with a variety of properties
- * @author Lukas Bonkowski, Matthew Neufeld, Owen Cooke
+ * @author Lukas Bonkowski, Matthew Neufeld, Owen Cooke, Sami Jagirdar
  * @see ItemAdapter
  */
-public class Item implements Serializable {
+public class Item implements Serializable, Parcelable {
```

Comment:
If it's parcelable does it need to be Serializable?

Reply:
It only needs to be for now because Owen is using Serializable to send a single item for his ViewItem and EditItem fragments. Later on, we can use Parcelable for those and remove Serializable for good